includes/class-wc-install.php

```diff
@@ -946,6 +946,14 @@ private static function get_schema() {
   slug varchar(200) NOT NULL DEFAULT '',
   PRIMARY KEY  (tax_rate_class_id),
   UNIQUE KEY slug (slug($max_index_length))
+) $collate;
+CREATE TABLE {$wpdb->prefix}wc_reserved_stock (
+	`order_id` bigint(20) NOT NULL,
+	`product_id` bigint(20) NOT NULL,
+	`stock_quantity` double NOT NULL DEFAULT 0,
+	`timestamp` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Suggest a rename here to "created" or "created_at"

Reply:
timestamp is used in other wc core tables - not we're also using this in blocks so a rename would cause issues for those existing users.